core/flow/rule_manager.go

```diff
@@ -80,6 +80,13 @@ func GetRules() []*FlowRule {
 	return rulesFrom(tcMap)
 }
 
+func ResetTcMap() {
```

Comment:
Use `LoadRules(nil)` instead?

Reply:
We may not need such `reset` functions actually :)
If necessary, it should be `ClearRules()`.

Reply:
ClearRules() seem make more sense.
What you see is what you get.